README.md

```diff
@@ -12,11 +12,20 @@ Copyright © 2017 FIXME
 
 Distributed under the Eclipse Public License either version 1.0 or (at
 your option) any later version.
-
-
-## NOTES
-- you need to provide a secrets.clj file with
+  
+## Getting Started
+This project uses re-natal which in turn wraps react-native (v0.41.1). You would need to follow their installation instruction: 
+- [re-natal](https://github.com/drapanjanas/re-natal)
+- [react-native](https://facebook.github.io/react-native/docs/getting-started.html)
+
+Once you have installed all the dependencies, run `lein prod-build` and then:
+- for development: `react-native run-android`. You should also check the instructions in *re-natal* regarding development in devices/emulators.
+- for debug/test version: `react-native bundle --dev false --platform android --entry-file index.android.js --bundle-output ./android/app/build/intermediates/assets/debug/index.android.bundle --assets-dest ./android/app/build/intermediates/res/merged/debug`. This will create an *apk* file which you can then install on your device without needing a connection to react-native development server.
+- for production - not supporter yet :(
```

Comment:
true. Will fix this once I get my hands in my computer :). You can also fix it if you want ;)